### PR TITLE
Pack nugets

### PR DESCRIPTION
**Pull request by QuanWanxx**
Pack Nugets in release stage.
Have create the release pipeline to push the Nuget packages to FhirAnalytics feed.

### CODE REVIEW

File: FhirToDataLake/azurepipelines/pack-nugets.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        "Microsoft.Health.Fhir.Synapse.DataWriter/Microsoft.Health.Fhir.Synapse.DataWriter.csproj",
+        "Microsoft.Health.Fhir.Synapse.HealthCheck/Microsoft.Health.Fhir.Synapse.HealthCheck.csproj",
+        "Microsoft.Health.Fhir.Synapse.SchemaManagement/Microsoft.Health.Fhir.Synapse.SchemaManagement.csproj"
+      )
+      
+      foreach ($project in $projects) {
+        dotnet pack "src/$project" -p:NuspecProperties="version=$(version)" -c Release -o $(Build.ArtifactStagingDirectory)/nupkgs
+      }
+
+- task: PublishBuildArtifacts@1
```
Comment (moria97) [inline FhirToDataLake/azurepipelines/pack-nugets.yml:21]: We will push the published nuget packages in release stage?

**Comment by BoyaWu10** [comment]
Not related with this PR but just curious why there's no [artifact](https://microsofthealthoss.visualstudio.com/FhirAnalytics/_build/results?buildId=24164&view=artifacts&pathAsName=false&type=publishedArtifacts) published in the build pipeline? Is this behavior the one we expected? It looks that we are publishing the function app in the [yaml](https://github.com/microsoft/FHIR-Analytics-Pipelines/blob/e89f4ef7f6b0f389f7f89ca4119134537817751d/FhirToDataLake/azurepipelines/ci.yml#L92).

**Comment by BoyaWu10** [comment]
> Not related with this PR but just curious why there's no [artifact](https://microsofthealthoss.visualstudio.com/FhirAnalytics/_build/results?buildId=24164&view=artifacts&pathAsName=false&type=publishedArtifacts) published in the build pipeline? Is this behavior the one we expected? It looks that we are publishing the function app in the [yaml](https://github.com/microsoft/FHIR-Analytics-Pipelines/blob/e89f4ef7f6b0f389f7f89ca4119134537817751d/FhirToDataLake/azurepipelines/ci.yml#L92).

I'm able to see the artifacts now. Just ignore this comment.

File: FhirToDataLake/azurepipelines/pack-nugets.yml (hunk 1 above)
Comment (QuanWanxx) [reply FhirToDataLake/azurepipelines/pack-nugets.yml:21]: Will push the nugets in another pipeline in [Release](https://microsofthealthoss.visualstudio.com/FhirAnalytics/_release?_a=releases&view=mine&definitionId=1) page.

File: FhirToDataLake/azurepipelines/pack-nugets.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+steps:
+- task: PowerShell@2
+  displayName: Pack Nuget packages
+  inputs:
+    targetType: inline
+    workingDirectory: FhirToDataLake
+    script: |
+      $projects = @(
+        "Microsoft.Health.Fhir.Synapse.Common\Microsoft.Health.Fhir.Synapse.Common.csproj",
```
[the remark this answers is not in the document]
Comment (QuanWanxx) [reply FhirToDataLake/azurepipelines/pack-nugets.yml]: Fixed, thanks.

**Review by moria97 — APPROVED** [review]

**Comment by QuanWanxx** [comment]
Also have updated the push Nugets pipeline in [Release](https://microsofthealthoss.visualstudio.com/FhirAnalytics/_release?_a=releases&view=mine&definitionId=1) page

**Pull request merged**